app/ui/browser-modern/views/browser/tabbar/tab.jsx

```diff
@@ -139,13 +136,13 @@ function mapStateToProps(state, ownProps) {
   const selectedPageId = PagesSelectors.getSelectedPageId(state);
   const selectedPageIndex = PagesSelectors.getPageIndexById(state, selectedPageId);
 
+  const isOverviewVisible = UISelectors.getOverviewVisible(state);
+
   return {
-    pageTitle: page ? page.title : '',
-    pageLocation: page ? page.location : '',
-    isActive: selectedPageId === ownProps.pageId,
-    isBeforeActive: selectedPageIndex === pageIndex + 1,
-    isAfterActive: selectedPageIndex === pageIndex - 1,
-    isOverviewVisible: UISelectors.getOverviewVisible(state),
+    tooltipText: page ? page.title || page.meta.title || page.location : '',
```

Comment:
Again, the same checking -- actually, should probably have this on the page model

```js
getTooltip() {
  return this.title || this.meta.title || this.location || '';
},
```